src/functions_linearalgebra.jl

```diff
@@ -26,6 +26,7 @@ Base.propertynames(named::NamedFactorization; kwargs...) = propertynames(parent(
 Base.iterate(named::NamedFactorization{L, T, <:LU}) where {L, T} = (named.L, Val(:U))
 Base.iterate(named::NamedFactorization{L, T, <:LQ}) where {L, T} = (named.L, Val(:Q))
 Base.iterate(named::NamedFactorization{L, T, <:SVD}) where {L, T} = (named.U, Val(:S))
+Base.iterate(named::NamedFactorization{L, T, <:Cholesky}) where {L, T} = (named.L, Val(:U))
```

Comment:
IIUC this correctly, this should return:

```
r = rand(4,4); Σ = r' * r
_l, _u = cholesky(Σ)
cholesky(Σ).L == _l
cholesky(Σ).U == _u
```

Compare with lu::

```
_l, _u = lu(Σ)
lu(Σ).L == _l # True
lu(Σ).U == _u # True
```

Reply:
This is working on julia 1 but failing on 1.{3/5} so I think i'm isunderstanding something here.

Reply:
I think that's fine. Though we should probably make this a minor release if you'd prefer to just pin the Julia version to 1.6 or higher.